css/at-rules/font-face.json

```diff
@@ -426,6 +426,57 @@
             }
           }
         },
+        "font_weight": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font-weight",
```

Comment:
This is not the right URL: The descriptor font-weight is not documented on MDN (yet). You need to remove this field.